### ISSUE TITLE
Overtime Hours field to be added "a la" Teleworking

### ISSUE THREAD

**Issue by jaragunde** [Igalia/phpreport#95]
[Trac import]
Reported by: agomez
Original date: Wednesday, 07 July 2010 16:44

In our project, we are facing the possibility to work "overtime hours" and invoice them with a different price than the regular ones.

Overtime hours is not possible (AFAIK) in PIN projects but they are in PEX ones and should be controlled in order to make the life easier to management.

My proposal would be to add a boolean field "a la" Teleworking in which it would be possible to specify if the time tracked was "overtime" or not.

**Comment by jaragunde** [comment]
[Trac import]
Comment by: eocanha
Original date: Friday, 15 October 2010 16:03

This bug, #118 and any other similar enhancement in the future could be solved with a single free text field named "tags".

**Comment by jaragunde** [comment]
[Trac import]
Comment by: agomez
Original date: Friday, 15 October 2010 16:09

Replying to [comment:1 eocanha]:
> This bug, #118 and any other similar enhancement in the future could be solved with a single free text field named "tags".
...

Please, not.

Tags are error prone. We need some boolean value for this, although tags could be a different enhancement too.

**Comment by jaragunde** [comment]
[Trac import]
Comment by: jaragunde
Original date: Friday, 15 October 2010 17:01

Replying to [comment:2 agomez]:
> Replying to [comment:1 eocanha]:
> > This bug, #118 and any other similar enhancement in the future could be solved with a single free text field named "tags".
> ...
> 
> Please, not.
> 
> Tags are error prone. We need some boolean value for this, although tags could be a different enhancement too.

I was also thinking on tags to solve these two issues. What about tags to be picked from a strictly defined set of options?

**Comment by jaragunde** [comment]
[Trac import]
Comment by: agomez
Original date: Friday, 15 October 2010 17:14

Replying to [comment:3 jaragunde]:
...
> I was also thinking on tags to solve these two issues. What about tags to be picked from a strictly defined set of options?
...

Just thinking from the performance point of view, I think tags would last a lot more than just searches taking into account boolean values in the database.

I think these two cases; overtime and on-site are important enough to have their own field, not discarding the idea of having tags too.

**Comment by jaragunde** [comment]
[Trac import]
Comment by: jaragunde
Original date: Tuesday, 15 January 2013 12:39

Better late than never: work started by mrobinson at branch ''mrobinson-onsite-field''. I'm taking care of integrating it.

**Comment by jaragunde** [comment]
[Trac import]
Comment by: jaragunde
Original date: Monday, 11 February 2013 19:47

Merged into `master`:

```
commit 645b018d46916e3f65579d99f0dc06ea44ca234b
Merge: e39110c 3f7aef7
Author: Jacobo Aragunde Pérez <jaragunde@igalia.com>
Date:   Mon Feb 11 15:59:21 2013 +0100

    Add boolean field "onsite" to Task entity (and to all associated forms and reports).
    
    (merge branch 'mrobinson-onsite-field' into installer)
```

**Comment by jaragunde** [comment]
[Trac import]
Comment by: jaragunde
Original date: Friday, 15 February 2013 12:34

Well, this is embarrassing. I've closed this ticket instead of #118 by mistake.

**Comment by dmtrek14** [comment]
This one feels like more of a process/project management issue than a real tool issue.
